src/ImportDefinitionsBundle/Export/Exporter.php

```diff
@@ -239,7 +247,13 @@ private function getObjectValue(Concrete $object, Mapping $map, $data, Definitio
         $value = null;
 
         if ($map->getGetter()) {
-            $setter = $this->getterRegistry->get($map->getSetter());
+            $getter = $this->getterRegistry->get($map->getGetter());
```

Comment:
This one is probably valid as well right? You shouldn't call getSetter on map here

Reply:
right